src/ArrayHelper.php

```diff
@@ -378,6 +378,141 @@ public static function setValue(array &$array, array|float|int|string|null $key,
         $array[self::normalizeArrayKey(array_shift($keys))] = $value;
     }
 
+    /**
+     * Appends a value to an associative array at the key path specified.
```

Comment:
```suggestion
     * Find array in an associative array at the key path specified and add passed value to him.
```

May be so? We do not add value to associative array...